htlcswitch/mailbox_test.go

```diff
@@ -536,6 +537,103 @@ func TestMailBoxDuplicateAddPacket(t *testing.T) {
 	})
 }
 
+// TestMailBoxDustHandling tests that DustPackets returns the expected values
+// after calling SetDustLimits for the local and remote dust sum.
+func TestMailBoxDustHandling(t *testing.T) {
+	t.Parallel()
+
+	ctx := newMailboxContext(t, time.Now(), testExpiry)
+	defer ctx.mailbox.Stop()
+
+	_, _, aliceID, bobID := genIDs()
+
+	// It should not be the case that the MailBox has packets before the
+	// dust limits are set. This is because the mailbox is always created
+	// *with* its associated link and attached via AttachMailbox, where the
+	// dust limits will be set. Even though the lifetime is longer than the
+	// link, the setting will persist across multiple link-creations.
```

Comment:
Ah I see, but when the mailbox is created, we still have access to the currently _static_ configs of each side of the channel?

Reply:
yup